dashboard/package.json

```diff
@@ -15,7 +15,8 @@
     "coverage": "npm run coverage:unit && npm run coverage:integration",
     "coverage:unit": "npm run test:unit && codecov",
     "coverage:integration": "npm run test:integration && codecov",
-    "dev": "nodemon ./server/index.js"
+    "dev": "nodemon ./server/index.js",
+    "generate-docs": "jsdoc --configure ../jsdoc.conf.json"
```

Comment:
We should maybe make this command similar to what we use in `newman` for consistency: `build-docs`

Ref: https://github.com/postmanlabs/newman/blob/develop/package.json#L33